### DISCUSSION TITLE
Aspire and Webassembly

### DISCUSSION THREAD

**Discussion by pnorriscd** [microsoft/aspire#1402] [Q&A]
Not sure if this is the right place to put this.

In VS2022 17.9 Preview 2, when you create a new Blazor webassembly project, it asks if you want Aspire orchestration. If you click yes, it creates a new solution where you can't debug the webassembly app. 

I haven't been able to find any way to debug webassembly run under app host.

Looking for any ideas or suggestions.

**Comment by DamianEdwards** [comment]
When you say "can't debug the webassembly app", what do you mean exactly? Can you launch the whole thing via the AppHost project but the debugger doesn't attach to the webassembly running the browser? Do you see an error? Does the app itself otherwise run correctly?

**Reply by pnorriscd** [reply]
All the breakpoints are a hollow circle and say The breakpoint will not currently be hit. No symbols have been loaded for this document.

<img width="340" alt="image" src="https://github.com/dotnet/aspire/assets/148796632/2bf18443-cea0-4b29-9c42-231b858a41e2">

**Comment by pnorriscd** [comment]
I can also add that if you don't go through orchestration and run the wasm app directly, breakpoints work fine.

**Reply by DamianEdwards** [reply]
Thanks, opened #1418 to track this.
